pennylane/_device.py

```diff
@@ -118,6 +118,9 @@ def __init__(self, wires=1, shots=0):
         self.num_wires = wires
         self.shots = shots
 
+        self.op_queue = []
+        self.ev_queue = []
```

Comment:
I think I would prefer calling them `ops_queue` and `expval_queue` to match the ops and expval modules/notation when defining quantum circuits. Can we make them private and expose them via an @property to make clear that we don't want plugins to only look but not mess with them? Would it maybe make more sense to set them to `None` instead of `[]`, or even better, raise an exception in the @property method if they are accessed while being "not set"? In this way the user will get a more useful error message instead of, e.g., accidentally looping over an empty array.

Reply:
Okay, will do!